tests/dag_processing/test_processor.py

```diff
@@ -656,49 +654,3 @@ def test_counter_for_last_num_of_db_queries(self):
                 session=session,
             ):
                 self._process_file(dag_filepath, TEST_DAG_FOLDER, session)
-
-
-class TestProcessorAgent:
```

Comment:
a) This was in the wrong file.
b) These tests were just of the sync mode controls, so not needed